### ISSUE TITLE
Can't reproduce the paper results

### ISSUE THREAD

**Issue by andreeabir** [wentaozhu/DeepLung#109]
Hi,
First of all, congratulations on the impressive results and thank you for providing the code for your paper. I am interested to reproduce the results of your paper, but after I carefully read the code and all the issues I still have some questions:
 I trained from scratch the detection model on the luna16 dataset, using 10-fold validation, but the results obtained are not close to yours. I used the provided script run_training.sh, without --resume 064.ckpt, and I've got the following results so far: 
fold9: 0.7486, fold8: 0.8135, fold7: 0.6071, fold6: 0.8172, fold5: 0.6044, fold4: 0.6685
I saw, that you recommended starting training from the existing 064.ckpt model, but I didn't find any documentation for it. On which data is trained the 064.ckpt model: on luna16 or on another dataset? If the former, on which subsets it is trained? Or if it is trained on the whole luna16 is it correct to use it for pretraining?

**Comment by 3learning12340** [comment]
@andreeabir hello, I am reproducing the results, we can communicate with each other. My email:perfect0insist@gmail.com and my QQ:1531415314. I am pleasured to solve the problem together.

**Comment by wentaozhu** [comment]
I am so sorry that I am too busy these days. It is the pretrained weights from Tianchi dataset. We also do experiments on Tianchi dataset as you can see in our paper https://arxiv.org/pdf/1801.09555.pdf. We split training, validation and test. The 064.ckpt is the weights from Tianchi.

**Comment by andreeabir** [comment]
Thank you so much for your quick response @wentaozhu . I tried searching for an answer in the previous issues and I found this one https://github.com/wentaozhu/DeepLung/issues/93 in which you said the 064.ckpt is trained on Luna16 dataset, not on the Tianchi dataset. Did I misunderstand something or did something change from then until now?
I would really like to do experiments on the Tianchi dataset, but I couldn't get it. I saw here: https://github.com/wentaozhu/DeepLung/issues/15 that you only made public the validation set from the Tianchi dataset. Can you please make public the training set from the Tianchi dataset? Thank you so much for your time!

**Comment by andreeabir** [comment]
Thank you @3learning12340 for your kindness! I would like to communicate and solve problems together.

**Comment by wentaozhu** [comment]
Oh. That is strange. It does not make sense. Maybe you can download Tianchi from https://tianchi.aliyun.com/competition/entrance/231601/information

And try to get the pretrained weight from training from Tianchi.
